.github/workflows/pylint.yml

```diff
@@ -0,0 +1,23 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
```

Comment:
I think we want to specify a version here, and add a .pylintrc so developers can know what they're on the hook for

Reply:
also, is this going to block all further development until the codebase is up to spec?  We also have the option of linting only those files in the changeset, which would allow us to lint incrementally.  Happy to play it either way, lmk what you think.

Reply:
Specified the version.

Let's agree to a standard to set, and follow that. For now, I see it as an assistant during review, not a requirement.